src/stepper/stepper.ts

```diff
@@ -0,0 +1,70 @@
+import { html, LitElement, nothing } from 'lit';
+import { property } from 'lit/decorators.js';
+import '../icon-button';
+import '../icons/plus';
+import '../icons/minus';
+
+export class Stepper extends LitElement {
+  @property({ type: String }) unit = '';
+
+  @property({ type: Boolean }) disabled = false;
+
+  @property({ type: Number }) step = 1;
+
+  @property({ type: Number }) min = -Infinity;
+
+  @property({ type: Number }) max = Infinity;
+
+  @property({ type: String, reflect: true }) size: 'small' | 'medium' =
+    'medium';
+
+  @property({ type: Boolean, reflect: true }) fullWidth = false;
+
+  @property({ type: Number, reflect: true }) value = 0;
+
+  render() {
+    const iconSize = this.size === 'small' ? 20 : 24;
+
+    return html`
+      <div class="stepper" part="stepper">
+        <tap-icon-button
+          .size=${this.size}
+          type="button"
+          part="decrease-button"
+          variant="ghost"
+          @click=${this.handleDecrease}
+          ?disabled=${this.disabled || this.value <= this.min}
```

Comment:
Please provide the `aria-*` attributes. https://www.w3.org/WAI/ARIA/apg/patterns/spinbutton/